boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-pinctrl.dtsi

```diff
@@ -59,4 +59,17 @@
 			nordic,drive-mode = <NRF_DRIVE_H0H1>;
 		};
 	};
+
+	/omit-if-no-ref/ pwm130_default: pwm130_default {
+		group1 {
+			psels = <NRF_PSEL(PWM_OUT0, 9, 2)>;
```

Comment:
This should be P9.00 for LED0 (P9.02 is LED2).

Reply:
It needs to be P9.02, because channel 0 in the PWM130 is on the P9.02 and it is first possible LED to be used with PWM signal.

Reply:
Okay. Then I think we need to use `pwm_led_2` instead of `pwm_led_0` and provide a comment that explains why LED0 cannot be used (similarly like it is for nRF54L15).